src/components/select/select.spec.js

```diff
@@ -153,6 +153,16 @@ describe('<md-select>', function() {
     expect($rootScope.myForm.select.$touched).toBe(true);
   }));
 
+  it('should not set touched if the element is disabled', inject(function($compile, $rootScope) {
```

Comment:
I think that this test is not testing the primary change in this PR. In the test blur event is triggered, while element without tabindex is not focusable, so it won't get blur event at all.